Debug-end-to-end-tests.md

```diff
@@ -1,105 +1,30 @@
 ## Table of contents
 
 * [Introduction](#introduction)
-* [Using the debugger](#using-the-debugger)
-* [Downloading screenshots](#downloading-screenshots)
-* [Downloading screen recordings](#downloading-screen-recordings)
+* [How to know the version of failing test suite](#how-to-know-the-version-of-failing-test-suite)
 
 ## Introduction
 
-Whenever you're debugging tests, you should create a debugging doc to document your work. This helps future contributors if they run into a similar bug in the future. If other people come in later to help you, they can also use the debugging doc to get up to speed on what you've already figured out. You can make a copy of [this template debugging doc](https://docs.google.com/document/d/1qRbvKjJ0A7NPVK8g6XJNISMx_6BuepoCL7F2eIfrGqM/edit?usp=sharing) to get started. Also check out the [[debugging docs wiki page|debugging-docs]].
+Whenever you are debugging tests, you should create a debugging doc to document your work. This helps future contributors if they run into a similar bug in the future. If other people come in later to help you, they can also use the debugging doc to get up to speed on what you've already figured out. You can make a copy of [this template debugging doc](https://docs.google.com/document/d/1qRbvKjJ0A7NPVK8g6XJNISMx_6BuepoCL7F2eIfrGqM/edit?usp=sharing) to get started. Also check out the [[debugging docs wiki page|debugging-docs]].
 
-There are many ways to go about debugging an E2E test, but here's one approach:
+There are many ways to go about debugging an E2E test, but here is one approach:
 
 1. Create a [[debugging doc|debugging-docs]].
 2. Look through the logs from the failed test to try and understand what went wrong. In particular:
 
    * Look for a line that says just `Killed`. This line indicates that some process was killed by the operating system for consuming too much memory. It's fairly safe to assume that the test failure was because of that process being killed.
    * Look for the stack trace and error message of the _first_ error. The trace might point you to where the error was thrown in the test code, and the message may help explain what went wrong.
 
-3. If you don't understand what the error message means, search for it online. Also look through the test code and `core/test/protractor_utils/action.js` to see if the error message (or something like it) is in our test code.
+3. If you don't understand what the error message means, search for it online. Also look through the test code and `core/test/webdriverior_utils/action.js` to see if the error message (or something like it) is in our test code.
```

Comment:
For readers at this point in the docs, the problem could still be in `protractor_utils/action.js` right?

Reply:
Right, updated.